tuiexporter/internal/sharedcomponent/sharedcomponent.go

```diff
@@ -8,9 +8,11 @@ package sharedcomponent // import "go.opentelemetry.io/collector/internal/shared
 
```

Comment:
Get the latest shared component: https://github.com/open-telemetry/opentelemetry-collector/blob/main/internal/sharedcomponent/sharedcomponent.go